docs/release-v1-perf-scale-benchmarks.md

```diff
@@ -0,0 +1,71 @@
+### Benchmarks
+
+This document shares some of the performance benchmarks observed as part of the v1 release.
+It will talk about what this means for user's perf and scale story.
+
+#### Background: How to interpret the numbers?
+
+KubeVirt is an extension for Kubernetes that includes a collection of custom resource definitions
+(CRDs) served by kubevirt-apiserver. These CRDs are managed by controllers. Due to the distributed
+nature of the system, understanding performance and scalability data becomes challenging without
+taking specific assumptions into account. This section aims to provide clarity on those assumptions.
+
+1. The data presented in the document is collected by `periodic-kubevirt-e2e-k8s-1.25-sig-performance`.
+1. The test creates 100 minimal VMIs, with a small pause of 100 ms between creation of 2 VMIs. The definition
+   of minimal VMIs can be found here https://github.com/kubevirt/kubevirt/blob/release-1.0/tests/performance/density.go#L273
```

Comment:
Can you use a permalink instead of a branch here?

https://github.com/kubevirt/kubevirt/blob/20f6caaba4108733a2c3f216e3247202929c1ef9/tests/performance/density.go#L273

Reply:
fixed